pages/samvera/manager_guide/hyrax_2.1/configuration-2.1.md

```diff
@@ -0,0 +1,89 @@
+
+Repository Administrators can create and define collection types for the repository. Collection types include Administrative Sets and User Collections. Detailed information about collections and collection types are available in the [Collections and Admin Sets](http://samvera.github.io/collection-overview.html) section of this website.
+
+## Settings: Pages
+
+![Pages](images\screenshots\admin-configuration-pages.png)
+
+The Pages panel allows administrators to edit the About Page, Help Page, Deposit Agreement, and Terms of Use. These are static pages in the repository. This is not quite a [WYSIWYG editor](https://en.wikipedia.org/wiki/WYSIWYG). The text will conform to the CSS styles used by your repository. Tip: to add hyperlinks, create the links in a rich text editor, like Microsoft Word, and paste the link into the Pages text box. There is currently no way to create a link directly in the text editor.
+
+## Settings: Content Blocks
+
+Similar to Pages, Content blocks are editable text elements for the homepage. You can add an Announcement, Marketing Text, or a note about a Featured Researcher using these blocks. The same limitations to text editing as noted in the Pages section apply here.
+
+![Announcement Content Block](images\screenshots\admin-configuration-contentblocks-announcements.png)
+
+The Announcement Text is useful for notifying end-users about any scheduled downtime or repository maintenance. It corresponds to:
+
+![Announcement on Homepage](images\screenshots\admin-configuration-announcement-homepage.png)
+
+![Marketing Text Content Block](images\screenshots\admin-configuration-contentblocks-marketing.png)
+
+The Marketing Text is a good place to add a tagline for your repository. It corresponds to:
+
+![Marketing Text on Homepage](images\screenshots\admin-configuration-marketing-homepage.png)
+
+![Featured Researcher Content Block](images\screenshots\admin-configuration-contentblocks-featured.png)
+
+The Featured Researcher content block allows you to add a note about any researcher you choose to highlight on the homepage of your repository.
+
+![Featured Researcher Text on Homepage](images\screenshots\admin-configuration-featured-homepage.png)
+
+## Settings: Features
+
+There are several features that are automatically enabled in Hyrax, but these can be toggled on and off from the Features panel:
+
+![Configuration Features](images\screenshots\admin-configuration-features-2.1.png)
+
+The highlighted option in the Actions column signifies the current selection. Click on the Off option for any feature you wish to disable for your repository. Here is a [complete list](https://github.com/samvera/hyrax/wiki/Feature-matrix) of features in Hyrax.
+
+## Workflow Roles
+
+The Workflow Roles panel will allow repository administrators to search for any registered user in the repository and assign them a role in an Administrative Set workflow. From the page, you can Assign Roles or view Current Roles.
```

Comment:
so this i think is only needed if people have "custom workflows" i'm not sure if it's recommended to add roles this way if you are using the "shipped" workflows.